frontend/src/styles/Moment.css

```diff
@@ -26,8 +37,49 @@
     position: relative;
     float: left;
     width:  400px;
-    height: 200px;
+    height: 320px;
     background-position: 50% 50%;
     background-repeat:   no-repeat;
     background-size:     cover;
 }
+
+/* Rotate all even images 2 degrees */
+.Moments-list li:nth-child(5n) .Moment-preview-container{
+  -webkit-transform: rotate(2deg);
+  -moz-transform: rotate(2deg);
+}
+/* offset every 3rd image's position */
+.Moments-list li:nth-child(5n+1) .Moment-preview-container{
+  -webkit-transform: none;
+  -moz-transform: none;
+  position: relative;
+  top: -5px;
+}
+/* Rotate every fifth image by 5 degrees and offset it */
+.Moments-list li:nth-child(5n+2) .Moment-preview-container{
+  -webkit-transform: rotate(5deg);
+  -moz-transform: rotate(5deg);
+  position: relative;
+  right: 5px;
+}
+/* Keep default rotate for every eighth, but offset it */
+.Moments-list li:nth-child(5n+3) .Moment-preview-container{
+  position: relative;
+  top: 8px;
+  right: 5px;
+}
+/* Keep default rotate for every eleventh, but offset it */
+.Moments-list li:nth-child(5n+4) .Moment-preview-container{
+  position: relative;
+  top: 3px;
+  left: -5px;
+}
+
+.Moments-list li .Moment-preview-container:hover {
+    cursor: pointer;
+    -webkit-transform: scale(1.2);
```

Comment:
The zoom is too much in my opinion. I'd change the value inside scale() to 1.05 (I tried it, looks much better). For this line and the line below.

Reply:
Okay no problem, I'm still going to be making small adjustments along the way. I was more concerned with the features functioning properly for now.